gradlew

```diff
@@ -1,7 +1,7 @@
-#!/usr/bin/env sh
+#!/bin/sh
```

Comment:
I realise that this is a generated script, but this is a regression that will cause many systems to break. The modern trend is to delegate to `env` to set the shell.